examples/argocd-keycloak-openshift.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   sso:
     provider: keycloak
+    verifyTLS: false
```

Comment:
Is it an intentional change?

Reply:
I added this for users who want to run the operator locally. I think I can put this in a better way, let me comment this line and add some description when this is needed.

Reply:
made the required changes.